bundler/lib/bundler/man/gemfile.5.ronn

```diff
@@ -526,6 +529,8 @@ remote versions that better match other requirements for the `gemspec` gem.
 
 ## SOURCE PRIORITY
 
+**This section describes a behavior of the feature deprecated in Bundler 1.13**.
+
```

Comment:
What do you think is the current logic in bullet point 3? While adding multiple global sources is just deprecated, not logic looks changed to me...

Reply:
This is describing the source priority logic, and bullet point 3 is the case with least priority, namely, if neither 1 nor 2 are met.

So, the standard wording of point 3 should be something like 

> * If neither 1 nor 2 are met, the global source will be used. If multiple global sources are specified, they will be prioritized from last to first, but this is deprecated since Bundler 1.13, so Bundler prints a warning and will abort with an error in the future.